EdFi.Ods.Utilities.Migration/Scripts/PgSql/02Upgrade/v34_to_v50/01 Structure/1100-AddNewTables50.sql

```diff
@@ -0,0 +1,610 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
Thought I added this comment but I forgot to come back and add it apparently.  This "uuid-ossp" extension was showing up in the schema diff as only being in the migrated database and not in the one built from source, and it doesn't seem to be used in this script.  Do we need this extension?

Reply:
I'm importing uuid-ossp module so that I can use uuid_generate_v1 function to generate default guid (https://www.postgresql.org/docs/9.1/datatype-uuid.html).  Is there another way to do this? 

Id                      UUID CONSTRAINT SurveyCourseAssociation_DF_Id DEFAULT uuid_generate_v1() NOT NULL,

Reply:
I would expect it to match the script generated by MetaEd, so something like:
`ALTER TABLE edfi.SurveyCourseAssociation ALTER COLUMN Id SET DEFAULT gen_random_uuid();`

If we're not approach uuid generation appropriately, that needs to be handled as a ticket to update the MetaEd generation, not a change for some of the tables as part of migration.